scrabbler/scrabbler.py

```diff
@@ -109,9 +102,18 @@ def show(self):
         """prints the board to terminal"""
         print(self.board)
 
-    def get_board(self):
-        """gets the board for this game"""
-        return self.board
+    def get_dictionary(self, saved_path, read_path):
+        if not Game.DICTIONARY:
+            # load a saved dictionary object or construct a new one
+            if os.path.exists(saved_path):
+                logger.info("loading saved dictionary file...")
+                Game.DICTIONARY = Dictionary.load_from_pickle(saved_path)
```

Comment:
Cache the dictionary at the class level so we only have to read it into memory once.